_posts/2020-12-11-docker-builds-gcr-github-actions.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title: Continuous docker builds with Github Actions
+author: ianwhitestone
+summary: Building & pushing a docker image to Google Container Registry with Github Actions
+comments: true
+image: images/dask-gcp/dask-gcp-bad-guy.png
+---
+
+{% include head.html %}
+
+<p align="center">
+    <img width="75%" src="{{ site.baseurl }}{% link images/docker-gcr-ga/docker-ga-gcr.png %}">
+</p>
+
+I've recently been using docker to package up my Python environment for deployment in a [Dask cluster on Google Cloud Platform]({% post_url 2020-09-29-single-node-dask-cluster-on-gcp %}) (GCP). If you're spinning up Dask clusters in an active codebase, you'll want to make sure you're regularly updating your docker image with the latest dependencies and code. To avoid the hassle of manually rebuilding your docker image whenever things change, I recommend creating a simple Github Actions workflow that rebuilds your docker image automatically. This can be configured to run whenever a new commit is pushed to master or a development branch, or any other [events that trigger a workflow](https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows).
```

Comment:
```suggestion
I've recently been using docker to package up my Python environment for deployment in a [Dask cluster on Google Cloud Platform]({% post_url 2020-09-29-single-node-dask-cluster-on-gcp %}) (GCP). If you're spinning up Dask clusters in an active codebase, you'll want to make sure you're regularly updating your docker image with the latest dependencies and code. To avoid the hassle of manually rebuilding your docker image whenever things change, I recommend creating a simple [Github Actions](https://github.com/features/actions) workflow that rebuilds your docker image automatically. This can be configured to run whenever a new commit is pushed to master or a development branch, or any other [events that trigger a workflow](https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows).
```